post_office/models.py

```diff
@@ -199,8 +199,9 @@ def prepare_email_message(self, connection=None):
                         att.is_inline = True
                         att.content_id = match.group(1)
                 for attachment in self.attachments.all():
-                    path = default_storage.save(os.path.join(settings.MEDIA_ROOT, attachment.file.name), attachment.file.name)  # should be the relative path
-                    msg.attachments.add([(path, attachment.name)])  # must add as a list of tuples to add a custom name
+                    path = os.path.join(settings.MEDIA_ROOT, attachment.file.name)  # should be the relative path
+                    with default_storage.open(path, "r") as content:
+                        msg.attachments.add([(content.read(), attachment.name)])  # must add as a list of tuples to add a custom name
```

Comment:
Does `add()` support attaching the content file the file? This is the same approach as the first commit you made and then you told me that add did not support this method of attachment